pan-domain-node/tsconfig.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
do we need all the comments in this file?

Reply:
It's the default you get with `tsc --init`. I'm not a big fan myself so I've removed them